xml/art_solution_automation.xml

```diff
@@ -30,13 +30,15 @@ https://confluence.suse.com/x/8IEnGw
   </info>
 
   <note>
-    <title>No Official &suse; Support</title>
+    <title>Note about Official &suse; Support</title>
     <para>
-      The referenced &salt; formulas install and configure &sles4sapreg;
-      and configure the &ha; cluster.
+      The referenced &salt; formulas install and configure &sles4sapreg; including
+      advanced configurations like SAP HANA System Replication setup and configure 
```

Comment:
Just the right placeholder: :wink: 

```suggestion
      advanced configurations like &hana; System Replication setup and configure 
```

Reply:
I knew there was something, but could not find hehe. Thanks @tomschr